watchtower/wtmock/client_db.go

```diff
@@ -110,7 +110,7 @@ func (m *ClientDB) CreateClientSession(session *wtdb.ClientSession) error {
 		Policy:           session.Policy,
 		SeqNum:           session.SeqNum,
 		TowerLastApplied: session.TowerLastApplied,
-		RewardPkScript:   session.RewardPkScript,
+		RewardPkScript:   cloneBytes(session.RewardPkScript),
```

Comment:
Is this diff a result of a bug that occurs if the bytes aren't copied?

Reply:
not that i am aware of, but i think it better mirrors the behavior of the real db. we do this in a few other places when copying byte slices, and figured i'd add it as well for reward pkscripts